src/Routing/Middleware/AssetMiddleware.php

```diff
@@ -167,8 +167,8 @@ protected function deliverAsset(ServerRequestInterface $request, SplFileInfo $fi
         return $response
             ->withHeader('Content-Type', $contentType)
             ->withHeader('Cache-Control', 'public,max-age=' . $maxAge)
-            ->withHeader('Date', gmdate(DATE_RFC7231, time()))
-            ->withHeader('Last-Modified', gmdate(DATE_RFC7231, $modified))
-            ->withHeader('Expires', gmdate(DATE_RFC7231, $expire));
+            ->withHeader('Date', gmdate('D, d M Y H:i:s \G\M\T', time()))
```

Comment:
I don't like replacing a constant with a hand-crafted format everywhere. If this truly no longer exists in PHP, should we consider adding it?

Reply:
You’re right.

Reply:
Maybe something that can be added to chronos?

It seems PHP will remove all [`DATE_*` constants](https://www.php.net/manual/en/datetime.constants.php#constant.date-rfc7231) so the best idea would be to define them ourselfes

Reply:
Only `SUNFUNCS_*` constants are deprecated.

https://github.com/php/doc-en/pull/4006

Reply:
That makes more sense

Reply:
OK, so the scope is a bit smaller:

> The DATE_RFC7231 and DateTimeInterface::RFC7231 constants have been deprecated.

Reply:
Thanks for the suggestion! For HTTP headers like Date and Last-Modified, the officially recommended format is **RFC 7231 / RFC 1123**, which is always in **GMT (D, d M Y H:i:s GMT)**. While `DateTimeInterface::ATOM` is ISO 8601 and machine-readable, strict HTTP compliance expects the RFC format. We could define a constant or helper to make it future-proof and easier to reuse, while keeping it compliant with HTTP standards.